privaterelay/settings.py

```diff
@@ -708,3 +714,6 @@ def _get_initial_middleware():
 if USE_SILK:
     SILKY_PYTHON_PROFILER=True
     SILKY_PYTHON_PROFILER_BINARY=True
+
+# Django 3.2 switches default to BigAutoField
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
I was curious if there was any reason to do this migration, unless you were hitting the limits of 32-bit indexes. This talks about some [theoretical improvements](https://serverfault.com/questions/160082/should-i-install-32-bit-database-or-64-bit-database), but [others find int8 slower than int4 slower when profiling](https://stackoverflow.com/questions/6633050/performance-difference-between-int4-and-int8-in-postgresql-on-64-bit-server). Most of those posts are about a decade old, so I'm assuming the perf issues have been worked out since then.

I'd start a new project on `BigAutoField`, but I'm not in a rush for Relay.